doc/release/1.17.0-notes.rst

```diff
@@ -129,6 +129,15 @@ the stream changes for any given seed is extremely small. If a 0 is encountered
 underlying generator, then the incorrect value produced (either ``np.inf``
 or ``np.nan``) is now dropped.
 
+``arr.writeable`` can be set to true slightly more liberal
```

Comment:
liberal -> liberally

Reply:
had missed this, changed it to read "``arr.writeable`` can be switched to true slightly more often", thought that might be simpler english.